bin/compute-classpath.sh

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script computes Spark's classpath and prints it to stdout; it's used by both the "run"
+# script and the ExecutorRunner in standalone cluster mode.
+
+# Figure out where Spark is installed
+#FWDIR="$(cd "`dirname "$0"`"/..; pwd)"
+FWDIR="$SPARK_HOME"
+
+echo "Using FWDIR: $FWDIR"
+
+#. "$FWDIR"/bin/load-spark-env.sh # not executable by defult in $SPARK_HOME/bin
+
+"$MAHOUT_HOME"/bin/mahout-load-spark-env.sh
```

Comment:
The scala version doesn't seem to be set using the export in mahout-load-spark-env.sh:

When testing against s source build of spark 1.5.1:  ```$ compute-classpath.sh
Using FWDIR: /home/andy/apache/spark-1.5.1
Failed to find Spark assembly in /home/andy/apache/spark-1.5.1/assembly/target/scala-
You need to build Spark before running this program.
```
Setting the scala version manually, the classpath is computed correctly though.  Maybe we should enforce Mahout's scala version anyways?